frontend_tests/node_tests/templates.js

```diff
@@ -221,16 +221,17 @@ function render(template_name, args) {
     html += "</table>";
 
     var buttons = $(html).find('.button');
+    var is_admin = page_params.is_admin;
+    if (is_admin) {
```

Comment:
How can you say you're exercising both sides of the conditional if you only ever set the value to `true`?  We should be testing something for the `false` case, such as asserting that things are actually hidden.